sql_test.py

```diff
@@ -0,0 +1,40 @@
+import mysql.connector
+from mysql.connector import errorcode
+
+# Obtain connection string information from the portal
+config = {
+    'host':'mictestdb.mysql.database.azure.com',
+    'user':'c_hawk7397',
+    'password':'m1(|<3yM0i_i53',
```

Comment:
We will definitely want to store this elsewhere asap.

Reply:
There is an azure key vault, though I'm not sure that will help when the db will likely not be hosted on Azure after this temp db. It is likely a good idea that we investigate how our db will be hosted so we can determine where to keep secrets such as this one.

Reply:
Maybe this would be good for current purposes? [Github environment secrets](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-an-environment)

Reply:
I think, we can table that since it would only really apply for a couple of weeks.